src/crawlee/storages/__init__.py

```diff
@@ -2,5 +2,14 @@
 from ._key_value_store import KeyValueStore
```

Comment:
I am also wondering about the structure, taking into account the new inheritance chain. Perhaps it could look something like this:

```
$ ls -1 src/crawlee/storages/
_base_storage.py
_dataset.py  
_key_value_store.py        
_request_queue.py
```

```
$ ls -1 src/crawlee/request_sources/
_request_list.py      
_request_source_tandem.py
_request_provider.py  
_request_source.py
```

Opinions?

Reply:
The new `request_sources` structure looks fine, I could do that :)